src/main/kotlin/study/racingcar/step4/InputView.kt

```diff
@@ -0,0 +1,18 @@
+package study.racingcar.step4
+
+class InputView {
+
+    private val SEPARATOR_COMMA = ","
+
+    fun inputCarNameList(): List<String> {
+        println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).")
+        val carNameList = readLine()!!.split(SEPARATOR_COMMA)
+        InputValidation().validationCarNameLength(carNameList)
+        return carNameList
+    }
+
+    fun inputRunCount(): Int {
+        println("시도할 횟수는 몇 회인가요?")
+        return readLine()!!.toInt()
```

Comment:
요 부분은 크게 상관없을 것 같지만 가급적이면 !! operator의 사용은 줄이는게 좋을 것 같아요.

아래 구현이 좋다는 것은 아니지만 Safe calls(?),  Elvis operator(?:)를 활용해보시면 좋을 것 같아서 추가해두었습니다.

```suggestion
        return readLine()?.toInt() ?: throw IllegalArgumentException("빈 값을 입력하셨습니다.")
```